lua/cinema/theater/services/sh_hls.lua

```diff
@@ -28,7 +28,7 @@ if CLIENT then
     end
 
     function SERVICE:LoadVideo(Video, panel)
-        panel:EnsureURL("https://swamp.sv/s/cinema/file.html")
+        panel:EnsureURL("https://swamp.sv/s/cinema/hls.html")
```

Comment:
I realized, for some reason, we weren't using hls.html for m3u8 playlists. I have no idea why this changed but yeah this fixes some instances. I don't think file.html handles playlist chunks or something because it starts at the beginning in some cases even though it will play the video.

Reply:
file.html uses Video.JS while hls.html uses hls.js, so that would make sense